src/strings/en-gb.json

```diff
@@ -565,6 +565,7 @@
     "OptionAllowAudioPlaybackTranscoding": "Allow audio playback that requires transcoding",
     "OptionAllUsers": "All users",
     "Option3D": "3D",
+    "OriginalLanguage": "Original Language",
```

Comment:
Undo the changes in the `en-gb.json` file. They must be made via Weblate. Otherwise it may cause merge conflicts.